.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
TIL about dependabot support for GH Actions. I'll add it to other projects as well.